addons/web/static/src/js/views/form/form_renderer.js

```diff
@@ -240,13 +240,13 @@ var FormRenderer = BasicRenderer.extend({
      * Quick edits the field we clicked on before the mode switch.
      *
      * @param {Object} quickEditInfo
-     * @param {string} quickEditInfo.id
+     * @param {string} quickEditInfo.fieldName
      * @param {any} quickEditInfo.extraInfo
      */
     quickEdit: function (quickEditInfo) {
-        const { id, extraInfo } = quickEditInfo;
+        const { fieldName, extraInfo } = quickEditInfo;
         const fieldWidget = this.allFieldWidgets[this.state.id]
-            .find(field => field[symbol] === id);
+            .find(field => field[symbol].startsWith(fieldName) && field.isFocusable());
```

Comment:
startsWith works but relies on an implementation detail... When we talked about this earlier I forgot about that. Is it ok to replace `field[symbol].startsWith(fieldName)` by `field.name === fieldName`?